src/user.js

```diff
@@ -38,9 +38,14 @@ class User {
   scopes() {
     let scopes = this.roles.map(role => 'assume:' + role);
     // add permission to manage scopes prefixed by the identity
-    ['create-client', 'delete-client', 'update-client', 'reset-access-token'].forEach(v => {
-      scopes.push('auth:' + v + ':' + this.identity + '/*');
-    });
+    let id = this.identity;
+    scopes.push(
+      `auth:create-client:${id}/*`,
+      `auth:delete-client:${id}/*`,
+      `auth:update-client:${id}/*`,
+      `auth:reset-access-token:${id}/*`,
+      `queue:create-artifact:login-identity/${id}/*`,
+    );
```

Comment:
I switched to using template literals so scopes can be added where the `<identity>/*` doesn't necessarily come at the end of the string - I can't decide if it is more readable or if it was better before.

Reply:
more :)